README.md

```diff
@@ -53,6 +53,87 @@ func main() {
 }
 ```
 
+DNS views and compatibility
+===========================
+
+DNS views is a means for NS1 to serve one set of data to one group of clients
+(e.g. internal employees), and different sets of data to other groups of
+clients (e.g. public internet). This has been largely exposed by allowing zones
+in the NS1 system to share the same FQDN, and allowing propagation to be
+controlled via ACLs and "views". For existing zones, and users that have no
+need for the added complexity of views, the default behavior is unchanged.
+However, it is important to understand that the requirement that an FQDN be
+unique within the network is removed in v3.x, and the ramifications of that.
+
+Views are available in API v3.x. For more information, see:
+https://help.ns1.com/hc/en-us/articles/360054071374
+
+COMPATIBILITY
```

Comment:
this seems like generally more information than needs to be in the SDK, this should be captured in API docs if it isn't. I don't think all of this belongs here. If you disagree, then all of the ddi-specific language needs to be changed.

Reply:
I think we need a short discussion of views and how view-aware code differs from older code, and the rest can be dropped.

Reply:
README is updated, PTAL